docs/kpod-ps.1.md

```diff
@@ -0,0 +1,119 @@
+% kpod(1) kpod-ps - Simple tool to list containers
+% Urvashi Mohnani
+% kpod-ps "1" "AUGUST 2017" "kpod"
+
+## NAME
+kpod-ps - Prints out information about containers
+
+## SYNOPSIS
+**kpod ps [OPTIONS] CONTAINER**
+
+## DESCRIPTION
+**kpod ps** lists the running containers on the system. Use the **--all** flag to view
+all the containers information.  By default it lists:
+
+ * container id
+ * the name of the image the container is using
+ * the COMMAND the container is executing
+ * the time the container was created
+ * the status of the container
+ * port mappings the container is using
+ * alternative names for the container
+
+**kpod [GLOBAL OPTIONS]**
+
+**kpod [GLOBAL OPTIONS] ps [OPTIONS]**
+
+## GLOBAL OPTIONS
+
+**--help, -h**
+  Print usage statement
+
+## OPTIONS
+
+**--all, -a**
+    Show all the containers, default is only running containers
+
+**--no-trunc**
+    Display the extended information
+
+**--quiet, -q**
+    Print the numeric IDs of the containers only
+
+**--format**
+    Pretty-print containers to JSON or using a Go template
+
+Valid placeholders for the Go template are listed below:
+
+| **Placeholder** | **Description**                                  |
+| --------------- | ------------------------------------------------ |
+| .ID             | Container ID                                     |
+| .Image          | Image ID/Name                                    |
+| .Command        | Quoted command used                              |
+| .CreatedAt      | Creation time for container                      |
+| .RunningFor     | Time elapsed since container was started         |
+| .Status         | Status of container                              |
+| .Ports          | Exposed ports                                    |
+| .Size           | Size of container                                |
+| .Names          | Name of container                                |
+| .Labels         | All the labels assigned to the container         |
+| .Label          | Value of the specific label provided by the user |
+| .Mounts         | Volumes mounted in the container                 |
+
```

Comment:
This is great.  When I'm back from PTO, maybe I'll go through some of the others and do the same.